print/print_test.go

```diff
@@ -26,23 +27,23 @@ func TestPrint(t *testing.T) {
   ! Other  ; Comment here
   ; Comment there
 
-2017/01/01 * (kode) Tx
+2017/1/1 * (kode) Tx
  Account1:Hello World        - 10.00 $
  Other                   (10.00 $ * 2)
 `,
 			`; comment
 ; second
-2016-01-01 = 2016-02-02 Tx
-    Account1:Hello World              10.00 $ @ 12.23 USD  ; Note 7 flames
+2016-01-01 = 2016-02-02 Tx  ; txn note
+    Account1:Hello World              $10.00 @ 12.23 USD  ; Note 7 flames
     Other                             (123 USD)  ; Note
 
 2016-01-01 ! Tx
-    Account1:Hello World              10.00 $ [2017-01-01]  ; Then comment
-    ! Other; Comment here
+    Account1:Hello World              $10.00 [2017-01-01]  ; Then comment
+    ! Other                           ; Comment here
     ; Comment there
 
 2017-01-01 * (kode) Tx
-    Account1:Hello World              -10.00 $
+    Account1:Hello World              -$10.00
     Other                             (10.00 $ * 2)
```

Comment:
so this flips the `$` from one side to the other?
are there some `commodity` clauses that can decide on which side and which format these things are?
would that be the right way to go?

Reply:
i'm not sure — I just know that for $ specifically it's uncommon in the US to write it on the right hand side. I have forgotten the details in the last 4 (!! time flies) years though.